### ISSUE TITLE
Inventory all BDII components and repositories using python 2 code

### ISSUE THREAD

**Issue by gwarf** [EGI-Foundation/bdii#16]
We should identify all BDII components and repositories that contain python code that should be updated for Python 3.

**Comment by gwarf** [comment]
Some first analyse provided by @enolfc:
```
./bdii/docs/conf.py
./glue-validator/tests/DomainTest.py
./glue-validator/tests/EntityTest.py
./glue-validator/tests/AdminDomainTest.py
./glue-validator/setup.py
./glue-validator/lib/egi-glue2/__init__.py
./glue-validator/lib/egi-glue2/types.py
./glue-validator/lib/egi-glue2/data.py
./glue-validator/lib/validator/ObjectTest.py
./glue-validator/lib/validator/lhcbTest.py
./glue-validator/lib/validator/__init__.py
./glue-validator/lib/validator/KnownIssues.py
./glue-validator/lib/validator/utils.py
./glue-validator/lib/validator/WLCGTest.py
./glue-validator/lib/validator/messages.py
./glue-validator/lib/validator/EntryTest.py
./glue-validator/lib/validator/EGIProfileTest.py
./glue-validator/lib/glue1/__init__.py
./glue-validator/lib/glue1/types.py
./glue-validator/lib/glue1/data.py
./glue-validator/lib/glue2/__init__.py
./glue-validator/lib/glue2/types.py
./glue-validator/lib/glue2/data.py
bdii/bin/bdii-update:#!/usr/bin/python
bdii/docs/conf.py:#!/usr/bin/env python3
glite-info-plugin-delayed-delete-status/src/glite-info-plugin-delayed-delete-status:#!/usr/bin/python
glite-info-static/sbin/glite-info-static:#!/usr/bin/python
glite-info-update-endpoints/bin/glite-info-update-endpoints:#!/usr/bin/python
glue-validator/bin/glue-schema-validator:#!/usr/bin/python
glue-validator/bin/glue-validator:#!/usr/bin/python
glue-validator/tests/DomainTest.py:#!/usr/bin/python
glue-validator/tests/EntityTest.py:#!/usr/bin/python
glue-validator/tests/AdminDomainTest.py:#!/usr/bin/python
glue-validator/setup.py:#!/usr/bin/env python
```

So the currently identified repositories are:
* https://github.com/EGI-Foundation/bdii
* https://github.com/EGI-Foundation/glue-validator
* https://github.com/EGI-Foundation/glite-info-static
* https://github.com/EGI-Foundation/glite-info-update-endpoints
* https://github.com/EGI-Foundation/glite-info-plugin-delayed-delete-status

We need to check the list of released packages to understand if there are some deprecated components that we could just archive.

**Comment by gwarf** [comment]
All python scripts should also go through python-modernize: https://github.com/mitsuhiko/python-modernize

**Comment by gwarf** [comment]
Closed by #24
